sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOIT.java

```diff
@@ -115,16 +116,55 @@ public static void setup() throws IOException {
             .get();
   }
 
+  @Test
+  public void testKafkaIOWithRunnerV2() throws IOException {
```

Comment:
Instead of naming the test after the runner, is it possible to name the test after the behavior that is being tested and use the configuration to ensure it runs on the appropriate runners? Is there a reason to have both this test and `testKafkaReadsAndWritesProperly`?

If so, it would be good to understand the difference through the test naming. From what I can tell the material difference is that this test uses a streaming pipeline to read where `testKafkaReadsAndWritesProperly` uses a batch one.

Reply:
That's correct. I'll do the renaming. Thanks for catching that!